index.js

```diff
@@ -16,6 +16,17 @@ if (!process.env.OPEN_QUAD_WORD) {
 
 let app = express();
 
+// Add JSONP support
+app.set('jsonp callback name', 'callback');
+
+// Middleware to handle JSONP requests
+app.use((req, res, next) => {
+  if (req.query.callback) {
+    res.setHeader('Content-Type', 'application/javascript');
+  }
```

Comment:
::question:: do you have to add [jsonp](https://expressjs.com/en/5x/api.html#res.jsonp) to wrap the response in the callback?

Reply:
I don't believe that I can use the express app.jsonp method since I'm using the proxy middleware. So I have to do it manually.

Reply:
I just added a test for jsonp and realized that it was already supporting it before these additions so I removed this commit.